### PR TITLE
Include workflow badges

### PR DESCRIPTION
**Pull request by stormofice**

### CODE REVIEW

**Review by StealWonders — APPROVED** [review]

File: .github/workflows/deploy-main-in-dev-env.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Docker Build & Deploy
+name: Deployment
```
Comment (tobiges) [inline .github/workflows/deploy-main-in-dev-env.yml:1]: Would name it Build & Deploy dev api

Comment (StealWonders) [reply .github/workflows/deploy-main-in-dev-env.yml:1]: Development Deployment

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Mensatt Backend
 
+[![Build & Deploy](https://github.com/mensatt/backend/actions/workflows/go.yml/badge.svg)](https://github.com/mensatt/backend/actions/workflows/go.yml)
```
Comment (StealWonders) [inline README.md:3]: Build & Test

File: README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+[![Deployment](https://github.com/mensatt/backend/actions/workflows/deploy-main-in-dev-env.yml/badge.svg)](https://github.com/mensatt/backend/actions/workflows/deploy-main-in-dev-env.yml)
```
Comment (StealWonders) [inline README.md:4]: Development Deployment

**Review by StealWonders — CHANGES_REQUESTED** [review]

**Review by tobiges — APPROVED** [review]

**Pull request merged**